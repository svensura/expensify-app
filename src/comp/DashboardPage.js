import React from 'react';

import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const DashboardPage = props => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default DashboardPage;