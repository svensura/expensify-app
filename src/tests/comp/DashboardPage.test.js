import React from 'react';
import { shallow } from 'enzyme';

import DashboardPage from '../../comp/DashboardPage';

test('SHOULD RENDER <DashboardPage /> CORRECTLY', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
});