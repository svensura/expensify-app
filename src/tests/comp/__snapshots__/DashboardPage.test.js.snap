// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SHOULD RENDER <DashboardPage /> CORRECTLY 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
